File: app/src/main/java/buggy/plandroid/com/plandroidbugs/Role.java
package buggy.plandroid.com.plandroidbugs;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Role {
    @JsonProperty(value = "uid") public String uid;

}

File: app/src/main/java/buggy/plandroid/com/plandroidbugs/Role.kt
package buggy.plandroid.com.plandroidbugs

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class Role(
    @JsonProperty(value = "uid")
    val uid: String
)